Dockerfile

```diff
@@ -37,9 +37,11 @@ RUN	echo deb http://llvm.org/apt/trusty/ llvm-toolchain-trusty main > /etc/apt/s
 # Packaged dependencies
 RUN apt-get update && apt-get install -y \
 	apparmor \
+	asciidoc \
 	aufs-tools \
 	automake \
 	bash-completion \
+	bsdmainutils \
```

Comment:
The docs are wrong. I'm changing the target to install-criu, and re-checking the dependency list. I'll update once I do.